tests/dashboard_utils.py

```diff
@@ -0,0 +1,78 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# isort:skip_file
+"""Utils to provide dashboards for tests"""
+
+import json
+
+from pandas import DataFrame
+from typing import Dict, Any
+
+from superset import db, ConnectorRegistry
+from superset.models.dashboard import Dashboard
+from superset.models.slice import Slice
+
+
+def create_table_for_dashboard(
+    df: DataFrame, tbl_name: str, database, schema: Dict[str, Any]
```

Comment:
nit: `database: Database`. I assume this will cause a circular import, which we can get around by doing

```python
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from superset.models.core import Database

def create_table_for_dashboard(
    df: DataFrame, tbl_name: str, database: "Database", schema: Dict[str, Any]
):
```
The same can be applied to the return type.

Reply:
Sure, thanks for spotting that. If there is no problem with circular import while running tests, do you think it still might need `TYPE_CHECKING`?

Reply:
No, if it works without it you should not use it.

Reply:
It should be ok. Thanks for suggestion.